app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsFragment.kt

```diff
@@ -70,11 +70,12 @@ class SettingsFragment : Fragment() {
 
         /**
          * Hide the Preference on selected layouts.
+         * @return [Preference] if visible otherwise null
```

Comment:
Please add *why* in the comment. I assume it is because of performance, but that is not stated here.

Reply:
makes sense, will do.